scripts/eval/yamls/copa.yaml

```diff
@@ -1,6 +0,0 @@
-icl_tasks:
```

Comment:
This yaml is used in the quickstart. Please either change the quickstart or leave this yaml